conda_env/cli/main_create.py

```diff
@@ -80,6 +80,8 @@ def execute(args, parser):
         # FIXME conda code currently requires args to have a name or prefix
         if args.name is None:
             args.name = env.name
+        else:
```

Comment:
Without this the environment won't get a name, because `requirements.txt` files cannot contain the name of the environment.